### PR TITLE
Removed duplicate lookups

### LINKED ISSUE #1035: Sinopia Editor pop-up message with "base material" QA lookup
See stage.sinopia.io "harvard:rt:test:single_lookup"

This template has a simple property with QA lookup for lc list "base material"  when opening rt the browser generates an infinite popup message:  "There are multiple configured list lookups for http://qa_test1"

Looking at the lookup config file: https://github.com/LD4P/sinopia_editor/blob/31bb926d80ece456b612e4465ea3befb830d9f7c/static/lookupConfig.json#L783  
it appears as thought 2 lists "base material" and "applied material" point to the same source list uri: https://id.loc.gov/vocabulary/mmaterial   Could this be the problem?

I also notice in the profile editor, when selecting the "value uri" from the QA list that when one tries to select "applied material" it automatically switches to "base material"

### PR DESCRIPTION
**Pull request by jcoyne**
Lookups must have a unique URI. Fixes #1035

### CODE REVIEW

File: static/lookupConfig.json [hunk 1]
```diff
@@ -930,11 +925,6 @@
     "uri": "https://id.loc.gov/vocabulary/msoundcontent",
     "component": "list"
   },
-  {
-    "label": "special playback characteristics",
-    "uri": "https://id.loc.gov/vocabulary/mplayback",
-    "component": "list"
-  },
```
Comment (ndushay) [inline static/lookupConfig.json]: this isn't a duplicate -- is it unused??

Comment (jcoyne) [reply static/lookupConfig.json]: It's a duplicate of https://github.com/LD4P/sinopia_editor/blob/dc995f042c1452872bbbeb077b2546826415d0a5/static/lookupConfig.json#L900

Comment (jcoyne) [reply static/lookupConfig.json]: See the issue for context: https://github.com/LD4P/sinopia_editor/issues/1035#issuecomment-514347925

Comment (ndushay) [reply static/lookupConfig.json]: gotcha - thanks.

**Pull request merged**